spec/features/merchants/items/index_spec.rb

```diff
@@ -13,12 +13,255 @@
+    describe 'As merchant, When I visit my items index page' do
+      describe 'Next to each item name I see a button to disable or enable that item.' do
+        describe 'When I click this button then I am redirected back to the items index' do
+          it 'And I see that the items status has changed' do
+
+            merchant_stephen = Merchant.create!(name: "Stephen's Shady Store")
+            merchant_roger = Merchant.create!(name: "Roger's Fancy Store")
+
+            item_toothpaste = merchant_stephen.items.create!(name: "Item Toothpaste", description: "The worst toothpaste you can find", unit_price: 4000 )
+            item_lamp = merchant_roger.items.create!(name: "Item Lamp", description: "You bet, it's a lamp", unit_price: 4000 )
+
+            visit merchant_items_path(merchant_stephen)
+
+            expect(item_toothpaste.enabled).to eq(false)
+
+            within("#item_#{item_toothpaste.id}") do
+            click_button("Enable")
+            end
+
+            expect(current_path).to eq(merchant_items_path(merchant_stephen))
+
+            within("#item_#{item_toothpaste.id}") do
+            expect(page).to have_button("Disable")
+            end
+
+          end
+        end
+      end
+    end
+  end
+
+    describe 'User Story 10 - Merchant Items Grouped by Status' do
+      describe 'When I visit my merchant items index page' do
+        describe 'I see two sections, "Enabled Items" and "Disabled Items"' do
+          it 'each Item is listed in the appropriate section' do
+
+            merchant_stephen = Merchant.create!(name: "Stephen's Shady Store")
+            merchant_roger = Merchant.create!(name: "Roger's Fancy Store")
+
+            item_toothpaste = merchant_stephen.items.create!(name: "Item Toothpaste", description: "The worst toothpaste you can find", unit_price: 4000 )
+            item_lamp = merchant_roger.items.create!(name: "Item Lamp", description: "You bet, it's a lamp", unit_price: 4000)
+
+            visit merchant_items_path(merchant_stephen)
+
+            within("#disabled_items") do
+            expect(page).to have_content(item_toothpaste.name)
+            expect(page).to_not have_content(item_lamp.name)
+            end
+
+            within("#enabled_items") do
+            expect(page).to_not have_content(item_toothpaste.name)
+            expect(page).to_not have_content(item_lamp.name)
+            end
+
+            within("#item_#{item_toothpaste.id}") do
+            click_button("Enable")
+            end
+            
+            within("#enabled_items") do
+            expect(page).to have_content(item_toothpaste.name)
+            end
+
+          end
+        end
+      end
+    end
+
+  describe 'User Story 11 - Merchant Item Create' do
+    describe 'When I visit my items index page I see a link to create a new item' do
+      describe 'When I click link, Im taken to a form that allows me to add item information' do
```

Comment:
It seems like these could be tested separately rather than a bunch of describe blocks leading to one big test. That way you can see where in the user story something is broken (is the link displaying, but can't be clicked on? Does the link route you to the right path? etc)